README.md

```diff
@@ -1 +1,64 @@
-# Ruby-Project-1
\ No newline at end of file
+# Ruby-Project-1 (bubble_sort)
+
+## Description
+There are following two methods in thhis repo.
+1. bubble_sort: Build a method #bubble_sort that takes an array and returns a sorted array. It must use the bubble sort methodology
+2. bubble_sort_by: create a similar method called #bubble_sort_by which sorts an array by accepting a block. Remember to use yield inside your method definition to accomplish this. The block will have two arguments that represent the two elements of the array that are currently being compared. The block’s return will be similar to the spaceship operator you learned about before: If the result of the block execution is negative, the element on the left is “smaller” than the element on the right. 0 means both elements are equal. A positive result means the left element is greater. Use the block’s return value to sort your array.
```

Comment:
Kindly clean up this paragraph. You don't need to add the instruction you received during its design. You are only supposed to explain what the method does. 🔴